maven-plugin/src/main/resources/help.yaml

```diff
@@ -24,9 +24,21 @@ help:
         desc: "Add demo data when setting up a platform server."
       - name: "Dfile"
         desc: "Path to installation.properties file for batch setup."
+      - name: "DdockerHost"
+        desc: "Docker host address"
+      - name: "DdbReset"
+        desc: "If 'true' reset the database"
+      - name: "DjavaHome"
+        desc: "Path to JDK Version"
+      - name: "Ddebug"
+        desc: "Port number for enabling remote debugging (defaults to '1044')."
```

Comment:
Is this really a port number or a flag to determine whether we are using debug mode?

Reply:
It is the debug port. https://github.com/openmrs/openmrs-sdk/blob/12d3ba529b963ab1e8b81e55ce195e883d8facb1/maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java#L433-L448